server/api/auth.py

```diff
@@ -58,6 +58,19 @@ def signup():
                 continue
             if request_dict.get(field, None) != None:
                 user_info[field] = request_dict[field]
+        # Create a full length address to get the geocode
+        address_info = [
+            user_info['street_address'],
+            user_info['city'],
+            user_info['state_or_province'],
+            user_info['country'],
+        ]
+        full_address = " ,".join(address_info)
```

Comment:
```suggestion
        full_address = ", ".join(address_info)
```
I think the way you have this now it would output "123 Fake Street ,Toronto ,ON ,Canada"

Reply:
Good catch, I just needed it to work with google maps lol